tests/integration/models/reddit/test_subreddit.py

```diff
@@ -229,14 +270,30 @@ def test_unignore_reports(self):
 
     def test_unread(self):
         self.reddit.read_only = False
-        with self.recorder.use_cassette(
-                'TestSubredditModeration.test_unread'):
+        with self.recorder.use_cassette('TestSubredditModeration.test_unread'):
             count = 0
             for item in self.reddit.subreddit('all').mod.unread():
                 assert isinstance(item, SubredditMessage)
                 count += 1
             assert count > 0
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_update_settings(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestSubredditModeration.test_update_settings'):
+            before_settings = self.subreddit.mod.settings()
+            self.subreddit.mod.update_settings(title='A')
+            assert self.subreddit.title == 'A'
+            after_settings = self.subreddit.mod.settings()
+
+            # Ensure that nothing has changed besides what was specified.
+            del before_settings['title']
+            del after_settings['title']
+            assert before_settings == after_settings
+
+            self.subreddit.mod.update_settings(title='B')
```

Comment:
While this resets the test for the next run, let's not worry about it since we only need it to be perfect once when we generate the cassette. It's easier to do that manually, than to try to make the test code reset the state for the future -- that got really annoying with the PRAW<4 tests.